soc/st/stm32/stm32mp13x/Kconfig.soc

```
@@ -0,0 +1,20 @@
+# STMicroelectronics STM32MP13 MPU line
+
+# Copyright (c) 2025 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_STM32MP13X
+	bool
+	select SOC_FAMILY_STM32
+
+config SOC_SERIES
+	default "stm32mp13x" if SOC_SERIES_STM32MP13X
+
+config SOC_STM32MP13_A7
```

Don't put core suffix in name if SoC has only one core:
```suggestion
config SOC_STM32MP13
```